src/tools/tools/file.fs

```diff
@@ -228,17 +228,17 @@ module  FileTools =
         let mutable message = ""
         if File.Exists( filename ) then 
             bExist <- true 
-            else
-                Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: create and write file '%s'" filename)
-                try 
-                    use fileStream = CreateFileStreamForWrite( filename )
-                    fileStream.Write( bytes, offset, len )
-                    fileStream.Close() 
-                with 
-                | e -> 
-                    message <- e.Message
-                    bExist <- true
-                Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: complete create and write file '%s' (exists = %b)" filename bExist)
+        else
+            Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: create and write file '%s'" filename)
+            try 
+                use fileStream = CreateFileStreamForWrite( filename )
+                fileStream.Write( bytes, offset, len )
+                fileStream.Close() 
+            with 
+            | e -> 
+                message <- e.Message
+                bExist <- true
+            //Logger.LogF(LogLevel.MediumVerbose,  fun _ -> sprintf "WriteBytesToFileConcurrentPCompare: complete create and write file '%s' (exists = %b)" filename bExist)
```

Comment:
I guess it's because there're compilation error about line 241 with VS2013?  We didn't get it in other places, since we have switched to VS2015.  It seems the new compiler relaxed the capture requirements of closure.  We probably should make it correct for VS2013 too.